src/main/java/com/epam/training/tasks/third/comparator/CubeObservableSorter.java

```diff
@@ -0,0 +1,78 @@
+package com.epam.training.tasks.third.comparator;
+
+
+import com.epam.training.tasks.third.entities.CubeObservable;
+import org.apache.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class CubeObservableSorter {
```

Comment:
well.. youi don't need it, first sort is an operation but you do it in different ways, like one operation and multiple implementations, but your multiple implementations are your comparators, so having a comparator sorting is always the same. Just create method sort in repository:
getSorted(Comparator) or similar